dali/operators/random/normal_distribution_op.h

```diff
@@ -78,19 +78,18 @@ class NormalDistribution : public Operator<Backend> {
 
 
   TensorListShape<> GetOutputShape(const workspace_t<Backend> &ws) {
-    DALI_ENFORCE(!(spec_.NumRegularInput() == 1 && IsShapeArgumentProvided(spec_)),
+    DALI_ENFORCE(!(this->spec_.NumRegularInput() == 1 && IsShapeArgumentProvided(spec_)),
                  make_string("Incorrect operator invocation. "
                              "The operator cannot be called with both Input and `shape` argument"));
-    if (spec_.NumRegularInput() == 1) {
+    if (this->spec_.NumRegularInput() == 1) {
       single_value_in_output_ = false;
       return ShapesFromInputTensorList(ws);
     } else if (IsShapeArgumentProvided(spec_)) {
       single_value_in_output_ = false;
       return ShapesFromArgument(ws);
-    } else {
-      single_value_in_output_ = true;
-      return ShapeForDefaultConfig(ws);
     }
+    single_value_in_output_ = true;
+    return ShapeForDefaultConfig(ws);
```

Comment:
Nitpick: I think it was more readable when it was in the `else` block.

Reply:
Yeah, but compiler complained...